src/treeview_commands.ml

```diff
@@ -43,7 +43,31 @@ let terminal_item =
   Vscode.TreeItem.set_command item command;
   item
 
-let items = [ select_sandbox_item; terminal_item ]
+
+  let construct_item =
+    let icon =
+      `LightDark
+        Vscode.TreeItem.LightDarkIcon.
+          { light = `String (Path.asset "collection-light.svg" |> Path.to_string)
+          ; dark = `String (Path.asset "collection-dark.svg" |> Path.to_string)
+          }
+    in
```

Comment:
That's not a great icon. Here is a suggestion, but feel free to chose [another one](https://code.visualstudio.com/api/references/icons-in-labels#icon-listing):
```suggestion
    let icon = `ThemeIcon (Vscode.ThemeIcon.make ~id:"tools" ()) in
```